owl/SOMA-OBJ.owl

```diff
@@ -2973,17 +2257,6 @@ The intention behind such a simple role is to have a way to represent, in a sche
                 </owl:allValuesFrom>
             </owl:Restriction>
         </rdfs:subClassOf>
-        <rdfs:subClassOf>
```

Comment:
here is a deleted axiom.
Probably it would be good to keep it and change *dirty object* to *physical object*